src/Components/Server/src/Builder/RazorComponentsEndpointBuilder.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// <see cref="IEndpointConventionBuilder"/> to add specific component extensions.
```

Comment:
This comment doesn't give much info regarding the class it's applied to - RazorComponentsEndpointBuilder.

Reply:
I've changed it a bit, but there are a couple of things.
1. This is not a type users use by name, it's just a handle from where we add additional extension methods.
2. It's an implementation detail, it's just public for technical reasons (so we can hang extension methods from it). The important thing to remark here is that is an IEndpointConventionBuilder